src/styles/sidebar/components/annotation-publish-control.scss

```diff
@@ -0,0 +1,119 @@
+@import './primary-action-btn';
+
+// A composite control consisting of an `annotation-publish-btn` and a
+// cancel button
+.annotation-publish-control {
+  display: flex;
+
+  &__cancel-btn {
+    @extend .btn--cancel;
+
+    margin-left: 5px;
+    font-weight: normal;
+
+    &__icon {
+      margin-right: 3px;
+      transform: translateY(10%);
+    }
+  }
+
+  // A split button with a primary submit on the left and a drop-down menu
+  // of related options to the right
+  .annotation-publish-control__btn {
+    $text-color: $color-seashell;
+    $default-background-color: $color-dove-gray;
+    $hover-background-color: $color-mine-shaft;
+    $h-padding: 9px;
+    $height: 35px;
+    $border-radius: 2px;
+    $arrow-indicator-width: 26px;
+
+    height: $height;
+    position: relative;
+
+    // Align the menu arrow correctly with the ▼ in the toggle
+    &-menu-arrow {
+      right: 5px;
+    }
+
+    // Make sure the menu content is wide enough to "reach" to the right-aligned
+    // menu arrow
+    &-menu-content {
+      min-width: 100%;
+    }
+
+    &-primary {
+      @include primary-action-btn;
+
+      // the label occupies the entire space of the button and
+      // shows a darker state on hover
+      width: 100%;
+      height: 100%;
+      text-align: left;
+      padding-left: $h-padding;
+      padding-right: $arrow-indicator-width + 8px;
+    }
+
+    // dropdown arrow which reveals the button's associated menu
+    // when clicked
+    &-dropdown-arrow {
+      position: absolute;
```

Comment:
I have a feeling that some of the CSS could be simplified, but I can't be sure until diving into the details. As long as the CSS is component-ized we can do that at a later time without unexpected changes in other parts of the UI. Let's leave it as-is for the current PR.